### PR TITLE
Document the usage of icons in templates

### PR DESCRIPTION
**Pull request by dtdesign**
n/t

### CODE REVIEW

File: docs/migration/wsc55/icons.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+If you ever need to alter the icons, such as applying a special color or transformation, you should wrap the icon in an element like `<span>` and apply the changes to that element instead.
+
+## Using Icons in Templates
+
+The new template function `{icon}` was added to take care of generating the HTML code for icons, including the embedded SVGs for brand icons.
+Icons in HTML should not be constructed using the actual HTML element, but instead always use `{icon}`.
+
+```smarty
+<button class="button">{icon name='bell'} I‘m a button with a bell icon</button>
```
Comment (TimWolla) [inline docs/migration/wsc55/icons.md]:
```suggestion
<button class="button">{icon name='bell'} I’m a button with a bell icon</button>
```

The correct apostrophe is “thick at the top, thin at the bottom” (matching the closing quote).

File: docs/migration/wsc55/icons.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+```
+
+Unless specified the icon will attempt to use a non-solid variant of the icon if it is available.
+You can explicitly request a solid version of the icon by specifying it with `type='solid'`.
+
+```smarty
+<button class="button">{icon name='bell' type='solid'} I‘m a button with a solid bell icon</button>
```
Comment (TimWolla) [inline docs/migration/wsc55/icons.md]:
```suggestion
<button class="button">{icon name='bell' type='solid'} I’m a button with a solid bell icon</button>
```

**Review by TimWolla — APPROVED** [review]
Contents LGTM. I can’t think of anything that needs to be added.

File: docs/migration/wsc55/icons.md (hunk 1 above)
Comment (dtdesign) [reply docs/migration/wsc55/icons.md]: Good catch. I sometimes miss the shift modifier and the difference between `‘` and `’` in my editor is unnoticable…

**Pull request merged**